Source/Core/Core/IOS/VersionInfo.h

```diff
@@ -37,5 +37,30 @@ struct MemoryValues
 };
 
 const std::array<MemoryValues, 41>& GetMemoryValues();
+
+enum Feature
+{
+  // Kernel, ES, FS, STM, DI, OH0, OH1
+  FEATURE_CORE = 1 << 0,
+  // SDIO
+  FEATURE_SDIO = 1 << 1,
+  // Network (KD, SO, SSL, NCD, Ethernet, Wi-Fi)
+  FEATURE_NET = 1 << 2,
+  // KBD
+  FEATURE_USB_KBD = 1 << 3,
+  // USB_HID v4
+  FEATURE_USB_HIDV4 = 1 << 4,
+  // SDv2 support
+  FEATURE_SDV2 = 1 << 5,
+  // New USB modules (USB, USB_VEN, USB_HUB, USB_MSC, OHCI0, USB_HIDv5)
+  FEATURE_NEW_USB = 1 << 6,
+  // EHCI
+  FEATURE_EHCI = 1 << 7,
+  // WFS (WFSSRV, WFSI, USB_SHARED)
+  FEATURE_WFS = 1 << 8,
+};
+
+int GetFeatures(u32 major_version);
```

Comment:
Hmm, that's true, it doesn't make it obvious _what_ is being returned... I'll change the return type to `Feature`